src/components/game_screen/Monster.js

```diff
@@ -1,15 +1,22 @@
-import React from "react";
+import React, {useEffect, useState} from "react";
 import monsterArray from "../../assets/monsters/monsterArray";
+import monsterAttackArray from "../../assets/monsters/monsterAttackArray";
 import MonsterName from "./MonsterName";
 
-const Monster = () => {
-  const randomMonster =
-    monsterArray[Math.floor(Math.random() * monsterArray.length)];
+const Monster = ({ isAttacking }) => {
+  const [Index, setIndex]= useState(null)
+  useEffect(()=>{
```

Comment:
capital I ?

Reply:
What are you going to do about it???? Eh?